### DISCUSSION TITLE
主打聲控、負離子淨化！台灣大攜手三洋推出首台智能清淨除濕機

### DISCUSSION THREAD

**Discussion by moonnight3503** [1101-GS4719-C/Python-Community-GS4719-C#804] [General]
現代機器都已智能化，智能家電對我們的日常生活已經不再陌生。台灣大哥大近來積極耕耘智慧家電市場，推出的除濕機主打具備負離子淨化及甲殼素抗菌濾網的雙效淨化設計、快速乾衣功能，以及支援Ok Google語音助理的聲控指令，只要簡短地說一句話，就能夠輕鬆簡單的關機，也可透過下載APP的方式，對環境濕度進行偵測及控制，我覺得這樣很方便，對於家裡眾多家電，一台家電都有專屬於自己的一支遙控器，家中可能有數十支遙控器，著實不太方便，如果能夠僅用一支手機操控，就能大幅減少家中遙控器的數量，去除太多遙控器的麻煩。

連結:https://3c.ltn.com.tw/news/51230

**Comment by bonny123865** [comment]
剛剛查了一下Pchome的價格，現在只要13900就可以買到欸!
不是我在業配，雖然價格可能感覺很高，但是能有先進的智能聲控，想想就值得!
可以我不是台灣大哥大的，台灣之星快跟進!!

**Comment by Pietra1226** [comment]
有智能家電真的很方便，常常家電忘記關，出門了還可以用手機遠端操控，節能環保。
他的生活情境不知道適用哪些情境

**Comment by evanchou1204** [comment]
我覺得智能家電很棒，但目前的Google助理感覺沒有到很有用，希望以後真的什麼事情都可以叫Google助理來做，比如做菜或是洗衣服什麼的。

**Comment by CPCpyt** [comment]
智能家具真的能夠在生活中對我們造成很大的助力，
尤其在都會區，空氣清淨的確是非常重要的課題。

**Comment by StevenWanglolz** [comment]
我覺得要實際實現一個智慧家庭還有點困難，除非是超級有錢，家具都是智慧型才有機會達成，我乖乖用手動的就好了 :(
